### ISSUE TITLE
App icon looks ugly on MIUI

### ISSUE THREAD

**Issue by SmartFinn** [termux/termux-app#249]
![screenshot_2017-02-03-13-22-56-282_com miui home_crop](https://cloud.githubusercontent.com/assets/819186/22589767/aa33b2c4-ea14-11e6-8c8c-230acc8d85a0.png)

**Comment by fornwall** [comment]
Hm, is this on MIUI Launcher (https://play.google.com/store/apps/details?id=com.i.miui.launcher)? But that one is at version 1.0.4 and doesn't look like that for me?

**Comment by SmartFinn** [comment]
No, I am using default launcher in MIUI 8. MIUI Launcher is not an official launcher.

**Comment by fornwall** [comment]
Aha, thanks!

**Comment by PwnicornDev** [comment]
That's MIUIs fault for not following Android design guidelines and trying to make the icons look like iOS.

**Comment by SmartFinn** [comment]
@PwnicornDev I agree with you, but if other apps looks awesome, and one app looks ugly, this a problem with that app.

@fornwall `stroke-width="1"` in ic_launcher.svg may solve my issue.

**Comment by fornwall** [comment]
@SmartFinn I agree, looks like the current icon doesn't comply with material design spec for launcher icons. Thanks for the help here!

Could you try installing a simple test app at https://fornwall.net/icon-test.apk and see if that icon looks ok (it's a simple empty app that does nothing, with a new Termux icon with tweaked size)?

**Comment by SmartFinn** [comment]
@fornwall  IconTest app has the same problem :confused:

![screenshot_2017-02-07-18-35-16-618_com miui home](https://cloud.githubusercontent.com/assets/819186/22700843/958eef20-ed64-11e6-9d83-ed1ddaf896ce.png)

**Comment by natebass** [comment]
I like the icon the way it is, it reminds me of the mac Terminal icon ![screen shot 2017-02-09 at 6 25 27 am](https://cloud.githubusercontent.com/assets/6665595/22787282/998b16ee-ee90-11e6-9d55-733891c22aa6.png)

@SmartFinn Note: You said it is a problem with Termux only; but I notice the icon (in your screenshot) for FolderSync  cuts off the letter e of "Lite". Ease up on poor termux :)

**Comment by natebass** [comment]
We could look into Adaptive Icons released in O to solve this. It could also help for circular icons.
https://developer.android.com/preview/api-overview.html#ai

**Comment by koppa** [comment]
This issue is already related to #830 and #552.
The icon looks also strange with the default samsung launcher.

![screenshot_20181023-115607](https://user-images.githubusercontent.com/595760/47353028-7d0c8100-d6bb-11e8-98ac-e8dd13d035e4.jpg)
